.github/workflows/helm_charts_scalar.yml

```diff
@@ -12,42 +12,68 @@ on:
     paths:
       - "charts/**"
 
+env:
+  HELM_VERSION: v3.5.2
+
 jobs:
   lint-chart:
     runs-on: ubuntu-latest
+    env:
+      PYTHON_VERSION: 3.7
+
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
 
-      - name: Fetch history for chart testing
-        run: git fetch --prune --unshallow
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: ${{ env.HELM_VERSION }}
 
-      - name: Run chart-testing (lint)
-        id: lint
-        uses: helm/chart-testing-action@v1.0.0-rc.2
+      - name: Set up Python
+        uses: actions/setup-python@v2
         with:
-          command: lint
-          config: .github/ct.yaml
+          python-version: ${{ env.PYTHON_VERSION }}
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.0.1
+
+      - name: Run chart-testing (lint)
+        run: ct lint --config .github/ct.yaml
 
   kubeval-chart:
     runs-on: ubuntu-latest
     needs: lint-chart
-    # Uses 1.16.4 since it doesn't support 1.16.13 in https://github.com/instrumenta/kubernetes-json-schema.
     strategy:
       matrix:
         k8s:
-          - v1.15.7
-          - v1.16.4
+          - v1.18.1
```

Comment:
The default schema location (https://github.com/instrumenta/kubernetes-json-schema) has no activity, so we can use a forked repo (https://github.com/yannh/kubernetes-json-schema) instead.

```
helm kubeval charts/scalardl -v 1.21.0 --schema-location https://raw.githubusercontent.com/yannh/kubernetes-json-schema/master
```

or

```
export KUBEVAL_SCHEMA_LOCATION=https://raw.githubusercontent.com/yannh/kubernetes-json-schema/master
helm kubeval charts/scalardl -v 1.21.0
```

Reply:
@superbrothers 

Thanks! fixed in f62d17e.

Is this forked repo often used when using kubeval?

https://github.com/instrumenta/kubernetes-json-schema/issues/26

Reply:
I don't know whether this repository is "often" used or not, but it looks there are no problems with use. Kubernetes Json schema is able to be generated by own, so it can be switch anytime.